pkgs/development/libraries/fbjni/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, fetchpatch
+, openjdk
```

Comment:
A nitpick.
```suggestion
, jdk
```

Reply:
These seem to be aliased, and i would rather use openjdk

```
[17:23:03] jon@jon-desktop /home/jon/projects/nixpkgs (arcanist-php80)
$ nix-build -A openjdk
/nix/store/m2gxaicfdpfbxpxs5j0j0qrih4m677y1-openjdk-17.0.3+7
[17:23:15] jon@jon-desktop /home/jon/projects/nixpkgs (arcanist-php80)
$ nix-build -A jdk
/nix/store/m2gxaicfdpfbxpxs5j0j0qrih4m677y1-openjdk-17.0.3+7
```